### PR TITLE
Api fetch from backEnd

### PR DESCRIPTION
**Pull request by tRomanowski**
switched my API fetch from the front end into the backend!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,73 +1,34 @@
-import { loadFromLocal, saveToLocal } from './utilityFunctions/localStorage';
-import { useEffect, useState } from 'react';
-
 import Form from './components/Form';
 import MainWrapper from './components/MainWrapper';
 import RecipeList from './components/RecipeList';
+import { useState } from 'react';
+
+// const fetcher = (...args) => fetch(...args).then(res => res.json());
```
Comment (FlyingMoe91) [inline src/App.js:6]:
```suggestion

```
I guess you can remove that, but maybe you left in on purpose. there are two more a few lines down

File: src/App.js [hunk 2]
```diff
@@ -1,73 +1,34 @@
...
     getNewRecipe();
   }
-  console.log(recipes);
-  console.log(missingIngredients);
-
   function handleDelete(id) {
     setRecipes(recipes.filter(recipe => recipe.id !== id));
   }
-
+  console.log(recipes);
```
Comment (FlyingMoe91) [inline src/App.js:31]:
```suggestion

```

**Review by FlyingMoe91 — COMMENTED** [review]
What should I say? It looks really good to me and makes sense, but I can't promise that there isn't sth to make better :D
Looks really Good 👍

File: src/App.js (hunk 2 above)
Comment (tRomanowski) [reply src/App.js:31]: Thanks Patrick for the review.

**Pull request merged**